.agent/skills/drn-utils/SKILL.md

```
@@ -228,11 +237,26 @@ var plainId = sourceKnownEntityIdUtils.ToPlain(entityId);
 > ID generation is automatically handled by `DrnContext` when SourceKnownEntities are saved.
 
+### Entity Creation-Date Filters
+
+`IEntityDateTimeUtils` filters `SourceKnownEntity.Id` by its 250ms Source-Known ID creation tick. Each date boundary maps to the minimum ID for that tick (payload bits cleared) and the maximum ID (all 31 app, instance, and sequence payload bits set).
+
+| Filter | Inclusive boundary | Exclusive boundary |
+|---|---|---|
+| `CreatedAfter` | `Id >= tick.Min` | `Id > tick.Max` |
+| `CreatedBefore` | `Id <= tick.Max` | `Id < tick.Min` |
+| `CreatedBetween` | `Id >= begin.Min && Id <= end.Max` | `Id > begin.Max && Id < end.Min` |
+| `CreatedOutside` | `Id <= begin.Max || Id >= end.Min` | `Id < begin.Min || Id > end.Max` |
```

markdownlint-cli2 (0.23.0)

[warning] Table column count
Expected: 3; Actual: 7; Too many cells, extra data will be missing

(MD056, table-column-count)